packages/Webkul/Shop/src/Resources/views/customers/account/wishlist/wishlist-product.blade.php

```diff
@@ -1,67 +1,67 @@
-<div class="account-item-card mt-15 mb-15">
-    <div class="media-info">
+<div class="flex gap-[65px] p-[25px] items-center border-b-[1px] border-[#E9E9E9]">
+    <div class="flex gap-x-[20px]">
+
         @php
             $image = $item->product->getTypeInstance()->getBaseImage($item);
+            $color = app('Webkul\Attribute\Repositories\attributeOptionRepository')->findOrFail($item->product->color);
```

Comment:
Remove this. Currently at this stage just leave it as it is.

Reply:
Done